app/views/admin/loans/show.html.slim

```diff
@@ -7,11 +7,14 @@
   = stylesheet_link_tag "admin/logs/logs", :media => "all"
   = stylesheet_link_tag "admin/calendar/calendar", :media => "all"
 
+= include_gon
+
 - content_for :scripts
   = javascript_include_tag "admin/forms/forms"
-  = javascript_include_tag "admin/timeline/select_all"
+  = javascript_include_tag "admin/forms/project_step_form"
   = javascript_include_tag "admin/expand"
   = javascript_include_tag "admin/calendar/calendar"
+  = javascript_include_tag "admin/timeline/select_all"
```

Comment:
I believe per-page JS is considered a no-no nowadays isn't it? Shouldn't it all be included in application.js so it gets minified and the number of requests is reduced?